### PR TITLE
Add Discord on README

### PR DESCRIPTION
**Pull request by liufuyang**
Using Discord as a public chat community is a very convenient way
of communications for developers, especially for developers located outside of
China.

For example Vue.js uses Discord as well (you could join by
https://chat.vuejs.org/)

Here to start we can just use a Discor never-expiring invitation link

https://discord.gg/6NN7rgr

(Please do not delete this invitation link from server settings after you login to Discord)

I will set PingCAP members to admin role after they join and shift
the Discord server ownership if needed later to PingCAP members

The effect looks like this after merging:
image

image

---

You are all welcome to join the Discord server from the following link and test it out before merging this 😄 :
https://discord.gg/6NN7rgr

### CODE REVIEW

**Comment by breeswish** [comment]
I think IRC, gitter and Slack would be more popular choices for international users. Discord is more likely a chat channel for gamers.

**Comment by liufuyang** [comment]
@breeswish have you checked out project like https://github.com/vuejs/vue which uses Discord as well? It's one of the hottest frontend UI component.

I used to think Discord is not for developers but now I feel it is actually pretty good.
See some discussions here:
https://www.reddit.com/r/discordapp/comments/6z9ajh/why_is_everyone_moving_from_slack_to_discord/
https://www.quora.com/Why-would-someone-use-discord-over-slack

In short, it is free, simple to login and create servers, also support (I think) all of the sweet things Slack gives you.

**Comment by liufuyang** [comment]
@breeswish However I think gitter might be a good choice as well. I am not a member of PingCAP so I cannot create a very official community channel there. Can someone of your team do that then I can update the PR into using gitter?

**Comment by coocood** [comment]
@liufuyang 
Thank you for your suggestion.
Maintaining an online chat community is a commitment.
We are not prepared to do it yet.

**Comment by liufuyang** [comment]
Understood. Thank you for your reply. I will close my PR then 😄

 If there is anything I can help with, please let me know. I don't mind put a few hours per week help organising if possible 🀄️

**Comment by coocood** [comment]
@liufuyang 
Sure.
I hope we can have another chance of cooperation in the future.😄
